src/components/back-to-top/_back-to-top.scss

```diff
@@ -9,7 +9,7 @@
     padding: 0.5em 0;
 
     > .ons-back-to-top__link {
-      padding: 0;
+      padding: 0%;
```

Comment:
You wouldn't add a percent sign here it would just be `0`. But I dont think this css is doing anything anyway so can be removed. There doesn't seem to be any padding being added to that element without this

Reply:
Aah, I accidentally left it in when there was padding. Now there is margin padding applied to another element.